Assets/Scripts/Bomb.cs

```diff
@@ -65,6 +66,7 @@ private void Blowup()
         StartCoroutine(CreateExplosions(Vector2.down));
         StartCoroutine(CreateExplosions(Vector2.left));
         GetComponent<SpriteRenderer>().enabled = false;
+        SoundManager.instance.PlayExplosion();
         Destroy(gameObject, .1f);

```

Comment:
I would say for the parameters, turn .1f into its own variable of some sort so that it's clearer what it is